src/app/client/projects/desktop/src/app/modules/offline/components/offline-help-videos/offline-help-videos.component.spec.ts

```diff
@@ -22,4 +59,27 @@ xdescribe('OfflineHelpVideosComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should initialize slide data', fakeAsync(() => {
+    const resourceService = TestBed.get(ResourceService);
+    resourceService.instance = resourceServiceStub.instance;
+    resourceService.frmelmnts = resourceServiceStub.frmelmnts;
+    component.ngOnInit();
+    tick(500);
+    expect(component.slideData).toBeDefined();
```

Comment:
add more assertions on what happens on ngoninit and what value will be set in slideData.